components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -7823,6 +7823,37 @@ def setInvertedAxis(self, inverted):
         (1, False, True): 'b',  # signed char
         }
 
+    @assert_pixels
+    def getHistogram(self, channels, binCount, globalRange=True,
```

Comment:
Changing in 5.3 a major job and with the rest of the tasks to be done I don't think this is realistic